server/src/main/java/io/crate/execution/engine/collect/sources/InformationSchemaIterables.java

```diff
@@ -167,34 +166,16 @@ public InformationSchemaIterables(final Schemas schemas,
             Stream.concat(
                 sequentialStream(PGTypes.pgTypes())
                     .filter(t -> t.typArray() != 0)
-                    .map(InformationSchemaIterables::typeToSignature)
+                    .map(x -> x.typReceive().asDummySignature())
                     .map(PgProcTable.Entry::of),
 
                 // Don't generate array_recv entry from pgTypes to avoid duplicate entries
                 // (We want 1 array_recv entry, not one per array type)
-                Stream.of(
-                    PgProcTable.Entry.of(
-                        Signature.scalar(
-                            "array_recv",
-                            DataTypes.INTEGER.getTypeSignature(),
-                            new ArrayType<>(DataTypes.UNDEFINED).getTypeSignature()
-                        )
-                    )
-                )
+                Stream.of(PgProcTable.Entry.of(Regproc.of("array_recv").asDummySignature()))
```

Comment:
the signatures return types are now all `undefined`, is it because they are not used?

Reply:
The signature is used in columns of the `pg_proc` table, but given that we don't have a real `array_recv`, or  `<type>_in` / `<type>_out` functions I think it should be alright to use a dummy value. 

If it turns out that it causes problems we'll probably have to introduce some kind of registry for the dummy pg streaming functions.